src/components/common/footer/Footer.jsx

```diff
@@ -14,7 +14,7 @@ const ADDRESS = <div className='missionStatement'>Nouri's Address:<br />7241 185
 
 export const Footer = () => {
   return (
-    <div>
+    <>
       <div className='links-container'>
```

Comment:
I think this links container can actually be put inside the footer tag since it's part of the whole footer. You will have to readjust the styling but the separation between areas of the page will be clearer.

Reply:
You would also be able to remove the fragment that surrounds the footer.

Reply:
Done!

Reply:
I'm seeing a new span tag added to replace the <> or <div> instead of all the contents being put under the footer element.

Reply:
Yes I used a span tag. If you want the elements totally I believe react will let me use an array to wrap the elements in [<Sections>,<Footer>] for example. I don't know how well that would work but worth a try.